Library/Formula/minizinc.rb

```diff
@@ -4,13 +4,8 @@ class Minizinc < Formula
   head "https://github.com/MiniZinc/libminizinc.git", :branch => "develop"
 
   stable do
```

Comment:
If there are no patches for the stable build you can take `url` and `sha256` out of the stable block and just do:
```ruby
  desc "Medium-level constraint modeling language"
  homepage "http://www.minizinc.org"
  url "https://github.com/MiniZinc/libminizinc/archive/2.0.6.tar.gz"
  sha256 "95b413c82f510e406f32bbb779fe1221a3b6bf2931854f61ca44bcefc0788f50"
  head "https://github.com/MiniZinc/libminizinc.git", :branch => "develop"
```